unison-run.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
 export HOME=/root
 cd /unison
+if [ -f /bin/unison-$UNISON_VERSION-fsmonitor ]; then
+  ln -s -f /bin/unison-$UNISON_VERSION-fsmonitor /bin/unison-fsmonitor
+fi
```

Comment:
To be honest, I dont remember at all.  Its been a while heh.  I think it was because at one point I had multiple versions  installed and wanted to use the right fsmonitor for the version that was starting, but I dont think thats the case anymore.